imports/ui/clients/searchClient.js

```diff
@@ -60,10 +60,12 @@ const debouncedSearch = _.debounce((query, options, callback) => {
   Meteor.call('searchClient', query, options, (err, res) => {
     if (err) {
       Alert.error(err);
-    } else {
+    } else { var i=2;
```

Comment:
there must be a more elegant way to implement this idea. How about returning an object with 2 fields: search results, a flag indicating if a button should be displayed?

Reply:
Also let's keep the `let` and `const` usage favorable to `var`